ext/groonga/extconf.rb

```diff
@@ -169,6 +169,7 @@ def configure_command_line(prefix)
   command_line << "--disable-document"
   command_line << "--disable-benchmark"
   command_line << "--without-cutter"
+  command_line << "--disable-groonga-httpd" unless ENV['RROONGA_WITH_HTTPD']
```

Comment:
Using `with_config` is better:

```ruby
command_line.concat(Shellwords.split(with_config("groonga-build-options") || ""))
```

```console
% gem install rroonga -- --with-groonga-build-options "--disable-groonga-httpd --other-option ..."
```

Reply:
Would that replace all the above `command_line` stanzas?

Reply:
Groonga's `configure` overrides the same options by later options.
For example, `./configure --disable-document --enable-document` enables document.

Reply:
Ok I think I'll just let you change the PR as see fit. :)

Reply:
Any news?

Reply:
Ah, sorry. I misunderstood your comment.
I waited your change.

I'll update later.

Reply:
Any news now? :)

Reply:
Now, I tried and I noticed that this branch is protected for this repository developers.
So I close this pull request and push to master directly.